tests/test_other.py

```diff
@@ -9811,7 +9811,7 @@ def test_mmap_and_munmap(self):
     self.do_other_test('mmap_and_munmap', emcc_args)
 
   def test_mmap_and_munmap_anonymous(self):
-    self.do_other_test('mmap_and_munmap_anonymous', emcc_args=['-s', 'NO_FILESYSTEM'])
+    self.do_other_test('mmap_and_munmap_anonymous', emcc_args=['-s', 'NO_FILESYSTEM', '-fsanitize=address', '-s', 'ALLOW_MEMORY_GROWTH=1'])
```

Comment:
how about adding an asan variation of this test (using parameterize)? there probably isn't any coverage lost by adding asan, but the current way makes it unclear if the test makes sense without asan too. so maybe a comment at least?